### PR TITLE
Completes the spelling of an administrative note search thing

### PR DESCRIPTION
**Pull request by Qwertytoforty**
## What Does This PR Do

Changes a spelling mistake from "The note search from X has complete." to "The note search from X has completed."

## Why It's Good For The Game

Make admin tools 1% better.

## Changelog
:cl:
fix: (Administration) Fixed spelling / grammar of when the note search finishes.
/:cl:

### CODE REVIEW

File: code/modules/admin/sql_notes.dm [hunk 1]
```diff
@@ -239,7 +239,7 @@
 			output += "<a href='?_src_=holder;shownoteckey=[index_ckey]'>[index_ckey]</a><br>"
 			CHECK_TICK
 		qdel(query_list_notes)
-		message_admins("The note search started by [usr.ckey] has complete. CPU should return to normal.")
```
Comment (AffectedArc07) [inline code/modules/admin/sql_notes.dm:242]: `Action Item A “is completed” is wrong, although “is being completed” or “is going to be completed” are proper verb forms.`

This was correct before.

Comment (S34NW) [reply code/modules/admin/sql_notes.dm:242]: But it's not `is complete` currently, it is `has complete`. **Has** is used with a past participle to form the perfect, pluperfect, and future perfect tenses. The verb **complete** that means **to finish** is combined with the past tense form **completed**.
So this change is correct.

File: code/modules/admin/sql_notes.dm [hunk 2]
```diff
@@ -239,7 +239,7 @@
...
+		message_admins("The note search started by [usr.ckey] has completed. CPU should return to normal.")
```
Comment (dearmochi) [inline code/modules/admin/sql_notes.dm:242]: Perhaps `Note search started by [usr.ckey] complete. CPU should return to normal.`?

**Review by Kyep — APPROVED** [review]

**Review by AffectedArc07 — APPROVED** [review]
Grammar use I can

**Pull request merged**